extract_verbs.sh

```diff
@@ -8,115 +8,140 @@ set -e
-		exit 1
-		;;
+    "bulgarian" | "bg")
+        LANGUAGE="bulgarian"
+        LANGCODE="bg"
+        ;;
+    "chinese" | "zh")
+        LANGUAGE="chinese"
+        LANGCODE="zh"
+        ;;
+    "dutch" | "nl")
+        LANGUAGE="dutch"
+        LANGCODE="nl"
+        ;;
+    "english" | "en")
+        LANGUAGE="english"
+        LANGCODE="en"
+    ;;
+    "estonian" | "et")
+        LANGUAGE="estonian"
+        LANGCODE="et"
+        ;;
+    "finnish" | "fi")
+        LANGUAGE="finnish"
+        LANGCODE="fi"
+        ;;
+    "french" | "fr")
+        LANGUAGE="french"
+        LANGCODE="fr"
+        ;;
+    "galician" | "gl")
+        LANGUAGE="galician"
+        LANGCODE="gl"
+        ;;
+    "german" | "de")
+        LANGUAGE="german"
+        LANGCODE="de"
+        ;;
+    "italian" | "it")
+        LANGUAGE="italian"
+        LANGCODE="it"
+        ;;
+    "latin" | "la")
+        LANGUAGE="latin"
+        LANGCODE="la"
+        ;;
+    "mongolian" | "mn")
+        LANGUAGE="mongolian"
+        LANGCODE="mn"
+        ;;
+    "polish" | "pl")
+        LANGUAGE="polish"
+        LANGCODE="pl"
+        ;;
+    "portuguese" | "pt")
+        LANGUAGE="portuguese"
+        LANGCODE="pt"
+        ;;
+    "russian" | "ru")
+        LANGUAGE="russian"
+        LANGCODE="ru"
+        ;;
+    "slovak" | "sk")
+        LANGUAGE="slovak"
+        LANGCODE="sk"
+        ;;
+    "spanish" | "es")
+        LANGUAGE="spanish"
+        LANGCODE="es"
+        ;;
+    "swahili" | "sw")
+        LANGUAGE="swahili"
+        LANGCODE="sw"
+        ;;
+    *)
+        echo "invalid language parameter: quitting"
+        exit 1
+        ;;
 esac
 
-# Form Wikipedia dump URL
+# download Wikipedia dump
 URL="http://download.wikimedia.org/"$LANGCODE"wiki/latest/"$LANGCODE"wiki-latest-pages-articles.xml.bz2"
-echo "Downloading dump from: $URL"
+echo "[CHECK] downloading Wikipedia dump to " $(pwd)
 wget $URL
-# Extract text
-if [ ! -d "extracted"]; then
+echo "[CHECK] downloaded Wikipedia dump to " $(pwd)
+
+# create directory to extract text
+if [ ! -d "extracted" ]; then
     mkdir extracted
+    echo "[CHECK] extracted directory created in " $(pwd)
+else
+    echo "[CHECK] extracted directory already exists in " $(pwd)
 fi
-bzcat "$LANGCODE"wiki-latest-pages-articles.xml.bz2 | scripts/lib/WikiExtractor.py -o extracted
-# Split extraction by article
-if [ ! -d "corpus"]; then
+
+# extract files (note: check system path to fact-extractor directory)
+bzcat "$LANGCODE"wiki-latest-pages-articles.xml.bz2 | fact-extractor/lib/WikiExtractor.py -o extracted
+echo "[CHECK] extracted files"
+
+# create directory to split extraction by article
+if [ ! -d "corpus" ]; then
     mkdir corpus
+    echo "[CHECK] corpus directory created in " $(pwd)
+else
+    echo "[CHECK] corpus directory already exists in " $(pwd)
 fi
-cat extracted/*/* | csplit --suppress-matched -z -f 'corpus/doc_' - '/</doc>/' {*}
+
+# split files and remove headers
```

Comment:
Id later planned to map Wikipedia article IDs to extracted document names instead of the sequential csplit numbers. but youre right. as it stands this will not work. will revert back..